api/nakadi-event-bus-api.yaml

```diff
@@ -888,6 +888,7 @@ definitions:
               Timestamp of creation of the Event generated by the producer.
             type: string
             format: date-time
+            example: '1996-12-19T16:39:57-08:00'
```

Comment:
From the guidelines:

"Timestamps are passed in UTC-related format via APIs and should be stored in UTC (i.e. without any offset information). Localization based on UTC should be done by the services that provide user interfaces, if required."

It's not clear for me that we should not accept anything that's not UTC. I would interpret that we should accept it and then manipulate it in order to store an UTC version of the value.

Wdyt?